iOS/WAStickersThirdParty/WAStickerPackResponse.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  StickerPackResponse.swift
+//  WAStickersThirdParty
+//
+//  Created by Poornima Sreedhar on 26/11/18.
+//  Copyright © 2018 WhatsApp. All rights reserved.
+//
+
+import Foundation
+let jsonurl = "https://api.myjson.com/bins/eg7i2"
+// Codable to Parse the JsonData into Model
+
+//MARK:WAStickerPackResponse
+//Response Model Class
+struct WAStickerPackResponse: Codable {
+    let iOSAppstoreLink: String = ""
+    let androidAppstoreLink: String = ""
+    let stickerPacks:[WAStickerPack]?
+    
+    enum CodingKeys: String, CodingKey {
+        case iOSAppstoreLink = "ios_app_store_link"
+        case androidAppstoreLink = "android_play_store_link"
+        case stickerPacks = "sticker_packs"
+
+    }
+    
+}
+
+// Sticker Pack
+struct WAStickerPack: Codable {
+    let identifier: String
+    let name: String
+    let publisher: String
+    let trayImage: String
+    let publisherWebsite: String?
+    let privacyPolicyWebsite: String?
+    let licenseAgreementWebsite: String?
+    let stickers:[WASticker]
+    enum CodingKeys: String, CodingKey {
+        case identifier
+        case name
+        case publisher
+        case trayImage = "tray_image_file"
+        case publisherWebsite = "publisher_website"
+        case privacyPolicyWebsite = "privacy_policy_website"
+        case licenseAgreementWebsite = "license_agreement_website"
+        case stickers
+        
+    }
+    
+    var imageData: ImageData?  {
```

Comment:
Maybe rename this to `trayImageData`. Currently it is a bit vague that this is the `ImageData` of the `trayIcon`.